prow/jobs/incubator/reconciler/reconciler.yaml

```diff
@@ -709,6 +709,51 @@ presubmits: # runs on PRs
             effect: NoSchedule
         nodeSelector:
             dedicated: "high-cpu"
+    - name: pre-main-reconciler-validate-go-mod
+      annotations:
+        pipeline.trigger: "pr-submit"
+      labels:
+        prow.k8s.io/pubsub.project: "sap-kyma-prow"
+        prow.k8s.io/pubsub.runID: "pre-main-reconciler-validate-go-mod"
+        prow.k8s.io/pubsub.topic: "prowjobs"
+        preset-kyma-guard-bot-github-token: "true"
+      run_if_changed: '^go\.mod$'
+      skip_report: false
+      decorate: true
+      cluster: untrusted-workload
+      max_concurrency: 10
+      branches:
+        - ^main$
+      extra_refs:
+        - org: kyma-project
+          repo: test-infra
+          path_alias: github.com/kyma-project/test-infra
+          base_ref: main
+      spec:
+        containers:
+          - image: "eu.gcr.io/kyma-project/test-infra/bootstrap:v20220427-9543160d"
```

Comment:
Have you thought about providing your own image with properly configured tools for this job with required packages?

Then you could just use in PodSpec:
```yaml
command: make
args:
  - validate-go-mod
```
without writing bash commands

Reply:
Hi @Ressetkk , thanks for this solution, do we have existing process to publish own image? If the workload is low, we can go this approach, but if our team need to setup and maintain the image release, then it's overkill for this task, this job is just simply want to run a python script, and bootstrap precompiled python, that's basically the reason choose it.

Reply:
@ruanxin We have no specific process to publish images and we have no strict rules for that on Prow for now. It's up to you on how you provide the image. You can push the image manually to the registry and update it here as you wish, and use the tag here.